Library/ViewModels/ThanksViewModelTests.swift

```diff
@@ -299,7 +297,6 @@ final class ThanksViewModelTests: TestCase {
       scheduler.advance()
 
       self.showRecommendationsProjects.assertValueCount(1, "Recommended projects emit, shuffled.")
```

Comment:
In addition to this, because below we removed `      self.showRecommendationsProjects.assertValues([[recommendedProject]])`, we can add it here to thoroughly cover _set_ of projects (if they are randomly sorted we can use a `set` comparison)

Reply:
I'm not sure I understand exactly. can you clarify your point a bit?

Reply:
Can skip this, I just meant that it seems like we don't check the individual projects returned from the response or the number of them. We do check the `[[recommendedProjects]]` in the removed test below, so thought we could add it here. If you have time, not a blocker or anything.